webservices/resources/reports.py

```diff
@@ -181,7 +181,10 @@ def build_query(self, entity_type=None, **kwargs):
             query = query.filter(
```

Comment:
same here. Remove `type` filter logic

Reply:
^will be done after users are notified due to breaking change as per our deprecation policy